sql/tsql/TSqlParser.g4

```diff
@@ -1959,7 +1959,7 @@ create_xml_index
 
 // https://msdn.microsoft.com/en-us/library/ms187926(v=sql.120).aspx
 create_or_alter_procedure
-    : ((CREATE (OR ALTER)?) | ALTER) proc=(PROC | PROCEDURE) procName=func_proc_name_schema (';' DECIMAL)?
+    : ((CREATE (OR ALTER| OR REPLACE)?) | ALTER) proc=(PROC | PROCEDURE) procName=func_proc_name_schema (';' DECIMAL)?
```

Comment:
```suggestion
    : ((CREATE OR (ALTER | REPLACE)?) | ALTER) proc=(PROC | PROCEDURE) procName=func_proc_name_schema (';' DECIMAL)?
```

Reply:
I think this will break a regular "create procedure". But "((CREATE (OR ( ALTER| REPLACE))?) | ALTER) proc..." is perhaps even better.

Reply:
Yes, sorry, you are right.

Reply:
I did an update in [b9b2747](https://github.com/antlr/grammars-v4/pull/2702/commits/b9b27477e72d886cbbde677439f98aecab3ce8a0) that should address both concerns :-)